streamlit-java/README.md

```diff
@@ -2,6 +2,8 @@
 
 This is an example template for a Snowflake Native Apps project which demonstrates the use of Java extension code and adding Streamlit code. This template is meant to guide developers towards a possible project structure on the basis on functionality, as well as indicate the contents of some common and useful files. 
 
+You cannot use this template as-is to create an application package and install an application. Please read below to understand the required intermediate steps.
```

Comment:
What is this trying to say exactly? It feels a bit confusing, i.e. this is a template but can't use it to create an application package?